src/svcutilcore/src/Resources/Strings.resx

```diff
@@ -208,13 +208,13 @@
     <value>The path to an assembly containing Service Contract types. Serialization types will be generated for all Xml Serializable types in each contract</value>
   </data>
   <data name="HelpXmlSerializerTypeGenerationSyntaxInput2" xml:space="preserve">
-    <value>Add the specified assembly to the set of assemblies used for resolving type references. (Short Form: /{0})</value>
+    <value>Add the specified assembly to the set of assemblies used for resolving type references. (Short Form: -{0})</value>
```

Comment:
Should "-" be "--" instead?

Reply:
@Lxiamail With this code change, we only want to support "--" before name (--out) and "-" before abbreviation (-o). So this short form should start with "-".